server/data/data-jpa/src/main/java/org/apache/james/user/jpa/JPAUsersRepository.java

```diff
@@ -98,23 +98,6 @@ public User getUserByName(Username name) throws UsersRepositoryException {
         }
     }
 
-    /**
-     * Returns the user name of the user matching name on an equalsIgnoreCase
-     * basis. Returns null if no match.
-     * 
-     * @param name
-     *            the name to case-correct
-     * @return the case-correct name of the user, null if the user doesn't exist
-     * @throws UsersRepositoryException
-     */
-    public String getRealName(Username name) throws UsersRepositoryException {
```

Comment:
could be used to get the case sensitive version of a username store in lower case?

Reply:
It's not part of the API

And a change for using similar mechanisms (but at the lib level) is under discussion here: https://github.com/linagora/james-project/pull/2887

I don't think this method adds value